app/services/disability_claim_service.rb

```diff
@@ -90,7 +90,7 @@ def claims_scope
 
   def create_or_update_claim(raw_claim)
     claim = claims_scope.where(evss_id: raw_claim['id']).first_or_initialize(data: {})
-    claim.update_attributes(data: claim.data.merge(raw_claim), successful_sync: true)
```

Comment:
I am starting to wonder if this `data.merge` might have caused some of the issues that Josh saw with data not clearing.